source/hosting/migrate-blob.txt

```diff
@@ -0,0 +1,199 @@
+.. _migrate-blob:
+
+Migrate to Your Own S3 Bucket
+=============================
+
+.. meta::
+  :description: Learn how to migrate your blob assets from Atlas App Services to your own AWS S3 bucket.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and AWS
+   for the most up-to-date and accurate information. The specific steps may vary
+   depending on the details of your project and the technologies
+   used.
+
+
+Migrate to S3
+-------------
+
+If you are solely using the MongoDB Atlas static hosting service as a blob store
+for static content and are not hosting a client application, follow the steps
+below to migrate from using Atlas Hosting to using your own S3 bucket.
+
+.. procedure::
+
+   .. step:: Create an S3 Bucket
+
+      Set up your own S3 bucket in your AWS account if you haven't already done
+      so. Configure the bucket settings according to your requirements,
+      including access permissions and encryption options.
+
+   .. step:: Migrate Existing Data
+
+      Migrate your existing data from MongoDB Atlas Hosting to your S3 bucket.
+      Depending on the amount of data and your specific requirements, this could
+      involve exporting data from MongoDB and uploading it to S3 using tools
+      like the AWS CLI or AWS SDK.
```

Comment:
I assume AWS CLI or AWS SDK handle the uploading to S3 part, but we don't provide any info here about how to export data from MongoDB. Should we recommend using the [App Services CLI](https://preview-mongodbkrollinsmdb.gatsbyjs.io/atlas-app-services/DOCSP-36783/cli/appservices-pull/) `appservices pull` command with the `--include-hosting` flag to get files out of Atlas? Is there another recommendation?

Reply:
Hmm. The CLI is probably the most straightforward way.